backend/config.py

```diff
@@ -68,7 +68,7 @@
 )
 from .connectors.usgs.source import NWISSiteSource, NWISWaterLevelSource
 from .connectors.wqp.source import WQPSiteSource, WQPAnalyteSource, WQPWaterLevelSource
-
+from .logger import Loggable
```

Comment:
I'm currently importing the logger as `from backend.logger import Loggable`. What's the advantage of using relative imports? Or is it essentially the same when thinking of it as a package?